screen-onboarding/src/main/java/com/ivy/onboarding/OnboardingDetailState.kt

```diff
@@ -0,0 +1,20 @@
+package com.ivy.onboarding
+
+import androidx.compose.runtime.Immutable
+import com.ivy.legacy.data.model.AccountBalance
+import com.ivy.legacy.datamodel.Category
+import com.ivy.legacy.utils.OpResult
+import com.ivy.wallet.domain.data.IvyCurrency
+import com.ivy.wallet.domain.deprecated.logic.model.CreateAccountData
+import com.ivy.wallet.domain.deprecated.logic.model.CreateCategoryData
+import kotlinx.collections.immutable.ImmutableList
+
+@Immutable
+data class OnboardingDetailState(
+    val currency: IvyCurrency,
+    val opGoogleSignIn: OpResult<Unit>?,
```

Comment:
The Google sign-in feature is removed. This is a dead code path - can you delete?